index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cmart | Login</title>
+    <link
+      rel="stylesheet"
+      href="https://aceeui.netlify.app/stylesheet/final.style.css"
+    />
+    <link rel="stylesheet" href="/styles/homepage.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="logo">
+        <a href="#" class="logo_link">
+          <i class="fas fa-birthday-cake fa-lg cake-logo"></i>
+        </a>
+        <h1 class="heading-title">Cmart</h1>
+      </div>
+      <ul class="list"></ul>
```

Comment:
If this element is not doing anything, please remove it.

Reply:
sure @ttahm3d  that is not doing anything removing that. :)